packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_ETH.sol

```diff
@@ -29,4 +30,47 @@ contract OVM_ETH is L2StandardERC20 {
             "ETH"
         )
     {}
+
+
+    /******************************
+     * Custom WETH9 Functionality *
```

Comment:
Although this scheme definitely works for Uniswap, are we sure it works for everyone else? I'm requesting that someone find other popular contracts that make use of WETH9 and confirm that this is sufficient for their needs.

Reply:
should work for others. 
Uniswap V2 (which anyone could deploy on the OVM if they wanted) and SushiSwap (which is a Uniswap V2 fork that AFAIK will deploy on the OVM when they can) both also use WETH9 a lot.
Maker uses it in a few spots such as their collateral adapters (WETH one here, called gem)
Yearn uses it for their ETH vaults, e.g. this random one